ebpfsvc/api.h

```diff
@@ -0,0 +1,12 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+void
+initialize_api_globals();
```

Comment:
```suggestion
initialize_api_globals(void);
```
For headers used only by C++ you need to include (void) which has a different meaning than () in C++.
(void) is strongly typed as 0 args, whereas () means anything is acceptable.  Same for line 12.